File: Online Judges/UVa/Solved/10696 - f91/f91.cpp
#include <iostream>

using namespace std;

long long f91(long long n) {
    if (n > 100) { return n - 10; }
    return f91(f91(n + 11));
}

int main() {

    long long n;
    cin >> n;
    while (n != 0) {
        cout << "f91(" << n << ") = " << f91(n) << endl;
        cin >> n;
    }

    return 0;
}

File: Online Judges/UVa/Solved/10696 - f91/f91.py
def f91(x):
    if (x <= 100):
        return f91((f91(x + 11)))
    return x - 10


n = int(input())
while (n != 0):
    print("f91({0}) = {1}".format(n, f91(n)))
    n = int(input())